app/Http/Controllers/User/UserController.php

```diff
@@ -155,7 +156,12 @@ public function update(Request $request, User $user): \Illuminate\Http\RedirectR
                 $image->move(public_path('/files/img/'), $filename);
             }
 
-            $user->image = $user->username.'.'.$image->getClientOriginalExtension();
+            $avatar = $user->username.'.'.$image->getClientOriginalExtension();
+
+            if ($user->image != $avatar) {
```

Comment:
I believe you should be able to safely use `!==` here. Correct me if I'm wrong.

Reply:
You're right, thanks for pointing it out.